certified-connectors/document-ai-konfuzio/apiDefinition.swagger.json

```diff
@@ -0,0 +1,623 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Document AI Konfuzio",
+    "termsOfService": "https://konfuzio.com/impressum/",
+    "contact": {
+      "name": "Helm & Nagel GmbH",
+      "email": "info@konfuzio.com",
+      "url": "https://help.konfuzio.com"
+    },
+    "license": {
+      "name": "Helm & Nagel GmbH"
+    },
+    "version": "v2",
+    "description": "Build custom Document AI to ocr, categorize and extract any document."
```

Comment:
@atraining Since this description would be published, instead of providing one line sentence, Please provide a detailed description of your connector.

Documentation - [Description of Connector](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)